roles/edpm_nova/tasks/configure.yml

```diff
@@ -101,6 +84,46 @@
   tags:
     - edpm_users
 
+- name: Sync nova config files
+  tags:
+    - configure
+    - nova
+  ansible.posix.synchronize:
+    src: "{{ lookup('env', 'HOME') }}/staging/"
+    dest: "{{ edpm_nova_config_dest }}"
+    recursive: true
+    delete: true
+  notify:
+    - Restart nova
+
+- name: Render nova config files
+  tags:
+    - configure
+    - nova
+  ansible.builtin.template:
+    src: "{{ item.src }}"
+    dest: "{{ edpm_nova_config_dest }}/{{ item.dest }}"
+    setype: "container_file_t"
+    mode: "0644"
+  loop:
+    - {"src": "config.json.j2", "dest": "config.json"}
+    - {"src": "nova-blank.conf", "dest": "nova-blank.conf"}
+    - {"src": "ssh-config", "dest": "ssh-config"}
+    # NOTE(gibi): This is unfortunate as we would like to avoid config
```

Comment:
nit we now also use this to enable tls.
it still does not change any of the sentiment that its unfortunate that this file need to exist and we would like to remove it in the future.
i would hope that eventually we do not need this for tls but we need to work that out

for now I'm fine with not updating the comment to mention that is used for enabling tls-e especially since it already references tls certificates.
that out of scope of this pr anyway as you are not changing this in this pr